model/validation.go

```diff
@@ -509,24 +515,35 @@ func ValidateLoadBalancing(lb *proxyconfig.LoadBalancing) (errs error) {
 func ValidateCircuitBreaker(cb *proxyconfig.CircuitBreaker) (errs error) {
 	if simple := cb.GetSimpleCb(); simple != nil {
 		if simple.MaxConnections < 0 {
-			errs = multierror.Append(errs, fmt.Errorf("circuit_breaker max_connections must be in range [0..]"))
+			errs = multierror.Append(errs,
+				fmt.Errorf("circuit_breaker max_connections must be in range [0..]"))
```

Comment:
Can you please rename fields to follow camelCase, e.g. maxConnections here? Proto allows both versions but we are trying to standardize on camelCase.